src/SamplesApp/UITests.Shared/Windows_Devices/RadioTests.xaml.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using Uno.UI.Samples.Controls;
+using Uno.UI.Samples.UITests.Helpers;
+using Windows.UI.Core;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+using Windows.Devices.Radios;
+
+namespace UITests.Shared.Windows_Devices
+{
+	[SampleControlInfo("Windows.Devices", "RadioTests", description: "Demonstrates use of Windows.Devices.Radio.GetRadiosAsyncTask")]
```

Comment:
```suggestion
	[Sample("Windows.Devices", description: "Demonstrates use of Windows.Devices.Radio.GetRadiosAsyncTask")]
```

Reply:
As CI reports `src\SamplesApp\UITests.Shared\Windows_Devices\RadioTests.xaml.cs(11,29): Error CS1739: The best overload for 'SampleAttribute' does not have a parameter named 'description`, I'm switching to old version,